pages/configuration/elasticsearch.rst

```diff
@@ -94,7 +94,7 @@ system defaults allow. **Set it to at least 64000 open file descriptors.**
 
 Graylog will show a notification in the web interface when there is a node in the Elasticsearch cluster which has a too low open file limit.
 
-Read about how to raise the open file limit in the corresponding `2.x <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/setup-configuration.html#file-descriptors>`__ / `5.x <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/file-descriptors.html>`__ documentation pages.
+Read about how to raise the open file limit in the corresponding `2.x <https://www.elastic.co/guide/en/elasticsearch/reference/6.6/setup-configuration.html#file-descriptors>`__ / `6.x <https://www.elastic.co/guide/en/elasticsearch/reference/6.6/file-descriptors.html>`__ documentation pages.
```

Comment:
you should fix this links correctly - as ES5 and ES6 are supported and your half fixed link writes ES2 but links to ES6 ...

Reply:
changed to wrong link to 2.3, all other to 6.7